tests/unit/src/utils/test_rlp.py

```diff
@@ -0,0 +1,113 @@
+import os
+from random import randint
+
+import pytest
+import pytest_asyncio
+from rlp import decode
+
+
+@pytest_asyncio.fixture
+async def rlp(starknet):
+    return await starknet.deploy(
+        source="./tests/unit/src/utils/test_rlp.cairo",
+        cairo_path=["src"],
+        disable_hint_validation=False,
+    )
+
+
+@pytest.fixture
+def data_to_list_long():
+    return list(
```

Comment:
I'd rather have

```
list(bytes.fromhex("80850430e2340082520894dc544d1aa88ff8bbd2f2aec754b1f1e99e1812fd01801ba0110d8fee1de53df0870e6eb599ed3bf68fb3f1e62c82dfe5976c467c97253b15a03450b73d2aef2009f026bcbf097a257ae7a37eb5d3b73dc0760aefad2b98e327"))
```

as it's shorter and char interpolation in your byte string is confusing

Reply:
also, I don't really understand what `data_to_list` means